actions/update_reporter.py

```diff
@@ -0,0 +1,13 @@
+from lib.base import BaseJiraAction
+
+__all__ = [
+    'UpdateReporterValue'
+]
+
+
```

Comment:
Would like to see unit tests for these new actions.

Reply:
I don't see unit tests for individual action that are already existing. could you please help me understanding, what are you expecting from those unit tests, since most of these actions are connecting to JIRA API and returning result ?